spec/v3/services/log/delete_spec.rb

```diff
@@ -100,7 +100,7 @@
     before do
       s3job.update_attributes(finished_at: Time.now)
       Fog.mock!
-      Travis.config.logs_options.s3 = { access_key_id: 'key', secret_access_key: 'secret' }
+      Travis.config.log_options.s3 = { access_key_id: 'key', secret_access_key: 'secret' }
```

Comment:
keychain looks okay - pro-keychain needs updating. I'll add `log_options` to pro-keychain, then remove `logs_options` after we've merged this and updated pro-api.